### PR TITLE
cuba , senegal , angola , EI salvador, cameroon, italy, ireland,leben…

### PR DESCRIPTION
**Pull request by maheshwaran-p**
## Summary

This pull request adds support for the following new currencies: Cuba, Senegal, Angola, El Salvador, Cameroon, Italy, Ireland, Lebanon, Mali, Malta, Madagascar, Togo, and Tonga.

### Changes Made

- Added new currencies to the currency list.
- Updated the country flags associated with the new currencies.

### CODE REVIEW

**Comment by MLZ91** [comment]
Why do you want to add Italy and Ireland for example? It is a currency list not a country list.

File: lib/src/currencies.dart [hunk 1]
```diff
@@ -1508,4 +1508,173 @@ List<Map<String, dynamic>> currencies = [
...
+    "decimal_digits": 2,
+    "number": 192,
+    "name_plural": "Cuban pesos",
+    "thousands_separator": ",",
+    "decimal_separator": ".",
+    "space_between_amount_and_symbol": true,
+    "symbol_on_left": true
+  },
+  {
+    "code": "XOF",
```
Comment (Daniel-Ioannou) [inline lib/src/currencies.dart:1525]: West African CFA Franc(XOF) it is already in the currency list

File: lib/src/currencies.dart [hunk 2]
```diff
@@ -1508,4 +1508,173 @@ List<Map<String, dynamic>> currencies = [
...
+    "decimal_digits": 2,
+    "number": 973,
+    "name_plural": "Angolan kwanzas",
+    "thousands_separator": ",",
+    "decimal_separator": ".",
+    "space_between_amount_and_symbol": true,
+    "symbol_on_left": true
+  },
+  {
+    "code": "USD",
```
Comment (Daniel-Ioannou) [inline lib/src/currencies.dart:1551]: United States Dollar (USD) it is already in the currency list

File: lib/src/currencies.dart [hunk 3]
```diff
@@ -1508,4 +1508,173 @@ List<Map<String, dynamic>> currencies = [
...
+    "decimal_digits": 2,
+    "number": 840,
+    "name_plural": "US dollars",
+    "thousands_separator": ",",
+    "decimal_separator": ".",
+    "space_between_amount_and_symbol": true,
+    "symbol_on_left": true
+  },
+  {
+    "code": "XAF",
```
Comment (Daniel-Ioannou) [inline lib/src/currencies.dart:1564]: Central African CFA franc(XAF) it is already in the currency list

**Review by Daniel-Ioannou — CHANGES_REQUESTED** [review]
Of the currencies you added, only the Tongan paʻanga, Malagasy ariary, Angolan kwanza, and Lebanese pound are missing from the list. Please remove the rest. This is a currency picker, not a country picker. The Euro is used by 26 countries, so we cannot add the euro 26 times."

**Pull request closed without merging**